src/net_processing.cpp

```diff
@@ -90,7 +90,7 @@ static_assert(INBOUND_PEER_TX_DELAY >= MAX_GETDATA_RANDOM_DELAY,
 /** Limit to avoid sending big packets. Not used in processing incoming GETDATA for compatibility */
 static const unsigned int MAX_GETDATA_SZ = 1000;
 /** Number of blocks that can be requested at any given time from a single peer. */
-static const int MAX_BLOCKS_IN_TRANSIT_PER_PEER = 16;
+static const int MAX_BLOCKS_IN_TRANSIT_PER_PEER = 2000; // YespowerSugar
```

Comment:
https://github.com/sugarchain-project/sugarchain/blob/7db380d865d2725d82d4bcd25a35659503934034/src/validation.h#L88-L93
/** IBD: which sets MAX_BLOCKS_IN_TRANSIT_PER_PEER to be same as MAX_HEADERS_RESULTS.
 *  Without this change, at least in Resistance the block headers download would get unnecessarily
 *  far ahead of the full blocks download, resulting in more work lost and redone in case the
 *  initial blocks download is interrupted and continued.
 *  The work loss is because we do not yet store cached PoWs on disk. */